app-utils/proot/autobuild/defines

```diff
@@ -3,3 +3,6 @@ PKGSEC=utils
 PKGDES="a powerful tool that can chroot, mount --bind, and binfmt_misc without privilege/setup"
```

Comment:
Capitalize the first word and remove extra adjectives.

Reply:
A tool for ...